capstoneteamb/capstone360
# Forms

some organization which will need to be resolved later, mostly just looking for functionality right now

to test:
clone the repo/checkout from origin if needed
checkout forms branch

run python3 app.py

go to the server address/forms/review

fill out the form and submit.

look over the code
in forms.py you can change the id of the user

in app.py you can change if the database is updated each time the app is launched (need to do this if updating the user)

send me a message on slack if you see any issues
